src/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <main>
+
+        <header>
+            <h1>Leaderboard</h1>
+        </header>
+    <div class="row">
+
+        <section class="scores">
+            <header>
+                <h2>Recent scores</h2>
+                <button class="button">Refresh</button>
+            </header>
+            <ul id="score-display">
+                <li>Name: 100</li>
+                <li>Name: 20</li>
+                <li>Name: 50</li>
+                <li>Name: 78</li>
+                <li>Name: 125</li>
+                <li>Name: 77</li>
+                <li>Name: 42</li>
+            </ul>
+        </section>
+        <section class="input-score">
+            <header>
+                <h2>Add your score</h2>
+            </header>
+            <form class="form">
+                <input type="text" name="name" class="input">
+                <input type="text" name="score" class="input">
+                <button type="submit" class="button">Submit</button>
```

Comment:
This button is on the right side of the form in the wireframe. Please follow the design of the wireframe as much as you can.